cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -200,6 +212,8 @@
     private KafkaAuthorization authorization;
     private Set<String> externalAddresses = new HashSet<>();
     private KafkaVersion kafkaVersion;
+    private boolean isJmxEnabled;
+    private boolean isAuthenticatedJmx;
```

Comment:
I am not a native English so it's upon you but as we have `isJmxEnabled`, doesn't it sound better `isJmxAuthenticated`?